### ISSUE TITLE
fix: 🤔 sd-video accessibility test

### ISSUE THREAD

**Issue by abudd1094** [solid-design-system/solid#483]
## Current behavior
The `sd-video` component is missing the standard `expect(el).to.be.accessible();` test found in most other components.  When attempting to implement it, it fails in Chromium in the combined test runner.  However, it passes successfully if run independently.
## Expected behavior
The test passes also in Chromium in the combined test runner.
## Steps to reproduce
At the time of implementation, we where unable to resolve this and excluded the test => reproduction not easy possible anymore.
## Technical Information
The test code follows:
``` 
it('passes accessibility test', async () => {
    const el: SdVideo = await fixture(variants.default);
    await expect(el).to.be.accessible();
});
```

Failing in combined runner:
![Image](https://github.com/solid-design-system/solid/assets/39494579/f23c7bb0-77b8-4d70-955d-c9a03f1955b3)

Succeeding independently:
![Image](https://github.com/solid-design-system/solid/assets/39494579/8c8ab0e0-646f-40ae-9e8d-53b960abd9e0)
## DoR
- [x] Item has business value
- [x] Item has been estimated by the team
- [x] Item is clear and well-defined
- [x] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant E2E tests (Features, A11y, Bug fixes) are created/updated
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch
